### PR TITLE
Update prettier to 2.3.1

### PR DESCRIPTION
**Pull request by sosukesuzuki**
| Q                        | A 
| ------------------------ | ---
| Patch: Bug Fix?          | Y
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  | Yes
| License                  | MIT

Fixed some behavior for assignments.

<a href="https://gitpod.io/#https://github.com/babel/babel/pull/13460"></a>

### CODE REVIEW

File: packages/babel-core/src/config/config-chain.ts [hunk 1]
```diff
@@ -717,8 +717,10 @@ function normalizeOptions(opts: ValidatedOptions): ValidatedOptions {
 function dedupDescriptors(
   items: Array<UnloadedDescriptor>,
 ): Array<UnloadedDescriptor> {
-  const map: Map<Function, Map<string | void, { value: UnloadedDescriptor }>> =
-    new Map();
+  const map: Map<
+    Function,
+    Map<string | void, { value: UnloadedDescriptor }>
+  > = new Map();
```
Comment (sosukesuzuki) [inline packages/babel-core/src/config/config-chain.ts:723]: :+1:

File: packages/babel-core/src/config/full.ts [hunk 2]
```diff
@@ -283,10 +283,14 @@ const makeDescriptorLoader = <Context, API>(
...
-const presetDescriptorLoader =
-  makeDescriptorLoader<Context.SimplePreset, PresetAPI>(makePresetAPI);
+const pluginDescriptorLoader = makeDescriptorLoader<
+  Context.SimplePlugin,
+  PluginAPI
+>(makePluginAPI);
+const presetDescriptorLoader = makeDescriptorLoader<
+  Context.SimplePreset,
+  PresetAPI
+>(makePresetAPI);
```
Comment (sosukesuzuki) [inline packages/babel-core/src/config/full.ts:294]: Maybe prev is better...

File: packages/babel-plugin-transform-typescript/src/namespace.ts [hunk 3]
```diff
@@ -68,9 +68,13 @@ function handleVariableDeclaration(
...
-    declarations.every((declarator): declarator is t.VariableDeclarator & {
-      id: t.Identifier;
-    } => t.isIdentifier(declarator.id))
+    declarations.every(
+      (
+        declarator,
+      ): declarator is t.VariableDeclarator & {
+        id: t.Identifier;
+      } => t.isIdentifier(declarator.id),
+    )
```
Comment (sosukesuzuki) [inline packages/babel-plugin-transform-typescript/src/namespace.ts:77]: Both new and prev are ugly..

**Review by nicolo-ribaudo — APPROVED** [review]

File: packages/babel-core/src/config/full.ts (hunk 2 above)
Comment (existentialism) [reply packages/babel-core/src/config/full.ts:294]: Yeah, I prefer previous

**Review by existentialism — APPROVED** [review]

**Pull request merged**